src/test/resources/application.conf

```diff
@@ -36,7 +36,7 @@ sqs {
 sns {
   endpoint = "http://localhost:9005"
   topic {
-    transform_engine_v2_in = "dHJlLWluLXRvcGljLWFybg==" # Base 64 decodes to this value tre-in-topic-arn
+    da_event_bus_arn = "da-eventbus-topic-arn"
```

Comment:
This should be base 64 encoded. Sorry didn't state that.

Reply:
Oh sorry. I didn't realise too, thanks.